lib/credentials/cognito_identity_credentials.js

```diff
@@ -66,27 +67,36 @@ AWS.CognitoIdentityCredentials = AWS.util.inherit(AWS.Credentials, {
 
   /**
    * Creates a new credentials object.
-   * @param (see AWS.CognitoIdentity.getId)
-   * @param (see AWS.STS.assumeRoleWithWebIdentity)
-   * @param (see AWS.CognitoIdentity.getOpenIdToken)
    * @example Creating a new credentials object
    *   AWS.config.credentials = new AWS.CognitoIdentityCredentials({
+   *
+   *     // either IdentityPoolId or IdentityId is required
    *     IdentityPoolId: 'us-east-1:1699ebc0-7900-4099-b910-2df94f52a030',
+   *     IdentityId: 'us-east-1:128d0a74-c82f-4553-916d-90053e4a8b0f'
+   *
    *     // optional, only necessary when the identity pool is not configured
    *     // to use IAM roles in the Amazon Cognito Console
    *     RoleArn: 'arn:aws:iam::1234567890:role/MYAPP-CognitoIdentity',
-   *     Logins: { // optional tokens, used for authenticated login
+   *
+   *     // optional tokens, used for authenticated login
+   *     Logins: {
```

Comment:
It might be good to expand on this list, since the parameters are being removed from the docs.